upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,194 @@
+          template:
+            metadata:
+              annotations:
+                iam.amazonaws.com/role: aws-service-operator
+              labels:
+                app: aws-service-operator
+            spec:
+              serviceAccountName: aws-service-operator
+              containers:
+              - name: aws-service-operator
+                image: awsserviceoperator/aws-service-operator:v0.0.1-alpha4
+                imagePullPolicy: Always
+                env:
+                - name: CLUSTER_NAME
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: CLUSTER_NAME
+                - name: REGION
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: REGION
+                - name: ACCOUNT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: ACCOUNT_ID
+                args:
+                  - server
+                  - --cluster-name=$(CLUSTER_NAME)
+                  - --region=$(REGION)
+                  - --account-id=$(ACCOUNT_ID)
+  customresourcedefinitions:
+    owned:
+    - name: cloudformationtemplates.service-operator.aws
+      version: v1alpha1
+      kind: CloudFormationTemplate
+      displayName: CloudFormationTemplate
+      description: Represents a resource for managing Cloud Formation Templates.
+    - name: dynamodbs.service-operator.aws
+      version: v1alpha1
+      kind: DynamoDB
+      displayName: DynamoDB
+      description: Represents a resource for managing a DynamoDB resource.
+    - name: ecrrepositories.service-operator.aws
+      version: v1alpha1
+      kind: ECRRepository
+      displayName: ECRRepository
+      description: Represents a resource for managing an ECR Repository.
+    - name: elasticaches.service-operator.aws
+      version: v1alpha1
+      kind: ElastiCache
+      displayName: ElastiCache
+      description: Represents a resource for managing an ElastiCache Engine.
+    - name: s3buckets.service-operator.aws
+      version: v1alpha1
+      kind: S3Bucket
+      displayName: S3Bucket
+      description: Represents a resource for managing a S3 Bucket.
+    - name: snssubscriptions.service-operator.aws
+      version: v1alpha1
+      kind: SNSSubscription
+      displayName: SNSSubscription
+      description: Represents a resource for managing a Simple Notification Service Subscription.
+    - name: snstopics.service-operator.aws
+      version: v1alpha1
+      kind: SNSTopic
+      displayName: SNSTopic
+      description: Represents a resource for managing a Simple Notification Service.
+    - name: sqsqueues.service-operator.aws
+      version: v1alpha1
+      kind: SQSQueue
+      displayName: SQSQueue
+      description: Represents a resource for managing a Simple Queue Service.
```

Comment:
s/Simple Queue Service/Amazon Simple Queue Service (SQS)/

Reply:
Resolved in the latest commit.